### DISCUSSION TITLE
Stanford STATS214 / CS229M: Machine Learning Theory - CSDIY.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#517] [Announcements]
# Stanford STATS214 / CS229M: Machine Learning Theory - CSDIY.wiki

CS自学指南

https://csdiy.wiki/en/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E8%BF%9B%E9%98%B6/CS229M/

**Comment by Skaiyin** [comment]
请问哪里可以找到作业呢
